zndraw/server/events.py

```diff
@@ -19,21 +45,35 @@
 
 def _webclients_room(data: dict) -> str:
     """Return the room name for the webclients."""
-    if "sid" in data:
-        return data["sid"]
-    return f"webclients_{data['token']}"
+    if isinstance(data, dict):
```

Comment:
If we had a base class, we could implement a custom `__contains__` method that removes this boilerplate.